Plantalytics/plantalytics-backend
# Be47 change email revisited

Added change email endpoint.

Endpoint
/email_change

Expected body:
{
"auth_token": "securitytokeninformation",
"new_email": "new_email_address@whatever.come"
}

Sends a notification email to the old email address about the change, so don't spam yourself!

To test you can use:
{
"auth_token": "securitytoken",
"new_email": "whatever@you.want.com"
}

Just be sure to reset the email back to plantalytics@gmail.com when done.

[ ] Hit endpoint to change email. Confirm the original (old) email address gets a notification
[ ] Hit endpoint to change email back to plantalytics@gmail.com and confirm your email from previous test gets a notification email.
[ ] Hit end point with invalid auth_token
[ ] Hit endpoint with missing attributes
NOTE: Email address checking ensures email is of the form something@other-things. It's pretty basic, but email addresses can't really be confirmed with regex. So, "plantalyticsatgmail.com" will be rejected. But "anything@anythingelse" will be accepted.
